### DISCUSSION TITLE
Add link to the solution directory

### DISCUSSION THREAD

**Discussion by fvbakel** [PlummersSoftwareLLC/PrimeView#24] [Ideas]
It would be nice if you could click on the solution number and that would navigate you to the corresponding directory in GitHub.

**Comment by rbergen** [comment]
I've opened #25 to add this.

**Reply by rbergen** [reply]
@fvbakel I'll have to await the review on #25 to merge it into and publish to production, but you can get a preview in staging: https://agreeable-mud-0b27ba210.azurestaticapps.net/.

**Reply by fvbakel** [reply]
@rbergen It works just as I imagined, thank you for the quick implementation!

**Reply by rbergen** [reply]
#25 was just merged. The feature should be available in PrimeView upon the next (re)load.
